mediathread/templates/dashboard/base_course_dashboard.html

```diff
@@ -36,13 +36,29 @@
         <div class="layout-horizontal-left">
             <nav aria-label="context" role="navigation">
                 <ul class="section-tabs">
+                    <li>
+                        <b>Preferences</b>
```

Comment:
the `<strong>` tag is generally used instead of `<b>` since HTML5, since it's more semantic... even though they pretty much do the same thing. https://www.w3schools.com/tags/tag_b.asp

Reply:
Ah! Got it. Made the change!